drivers/pci/pci.c

```diff
@@ -157,6 +159,46 @@ void rtc_set_checksum(int range_start, int range_end, int cks_loc)
 	CMOS_WRITE(((sum >> 0) & 0x0ff), cks_loc+1);
 }
 
+/* SMSC LPC47M157 Super I/O functions */
+/* pin and register compatible with LPC47M192 */
+void LpcEnableConfiguration(int Enable)
+{
+	IoOutputByte(0x2e, Enable ? 0x55 : 0xAA);
+}
```

Comment:
For code clarity I feel like it's better to split this into `LpcBeginConfiguration` and `LpcEndConfiguration` (or similar) to show the importance and block-nature of this.